grid.py

```diff
@@ -428,27 +450,24 @@ def populate_lines(self, orientation: str) -> None:
         Args:
             orientation (str): The orientation of the lines being divided (expects "rows" or "columns").
         """
-        half_grid: int = len(self._grid) // 2 + 1
-        for line in range(half_grid):
-            if line % 2 == 0:
-                usable_spaces: List[List[int]] = self.find_usable_spaces(line, orientation)
+        half_grid: int = len(self._grid) // 2
+        for line in range(half_grid + 1):
+            if line < half_grid:
+                usable_spaces: List[List[int]] = self.find_usable_spaces(line, orientation, half_grid)
                 for usable_space in usable_spaces:
                     first_space, last_space = (usable_space[0], usable_space[1])
                     self.create_word_divisions(first_space, last_space, line, orientation)
             else:
                 if orientation == "rows":
-                    self.create_alternating_divisions(line)
-                    if line != half_grid - 1:
-                        # Mirror in the bottom rows
-                        self.create_alternating_divisions(len(self._grid) - 1 - line)
-
-    def create_alternating_divisions(self, row: int) -> None:
+                    self.place_center_divider_randomly(half_grid)
+                
+    def place_center_divider_randomly(self, half_grid: int) -> None:
```

Comment:
Can we change the name to something closer to the description? The function name is quite vague but the description is on point. Maybe words from the description can be borrowed? Suggestion for clarity